tests/integration-tests/conftest.py

```diff
@@ -377,6 +377,35 @@ def api_client(region, api_server_factory, api_uri):
         yield api_client_instance
 
 
+@pytest.fixture(scope="class")
+def images_factory(request):
+    """
+    Define a fixture to manage the creation and destruction of images.
+
+    The configs used to create clusters are dumped to output_dir/images_configs/{test_name}.config
+    """
+    factory = ImagesFactory()
+
+    def _image_factory(image_id, image_config, region):
+        image_config_file = _write_image_config_to_outdir(request, image_config)
+        image = Image(
+            image_id="-".join([image_id, request.config.getoption("stackname_suffix")])
+            if request.config.getoption("stackname_suffix")
+            else image_id,
+            config_file=image_config_file,
+            region=region,
+        )
+        result = factory.create_image(image)
+        if "Build image started successfully." not in result:
```

Comment:
What about searching for `BUILD_IN_PROGRESS`? I think it's more stable than searching for a string that we can change in future and that we'll remove when switching to pure json output.

Reply:
good call, done